beacon-chain/operations/attestation.go

```diff
@@ -137,6 +138,23 @@ func (s *Service) AttestationPoolNoVerify(ctx context.Context) ([]*ethpb.Attesta
 	return atts, nil
 }
 
+// AttestationsBySlotCommittee returns the attestations from the attestations pool filtered
+// by slot and committee index.
+func (s *Service) AttestationsBySlotCommittee(ctx context.Context, slot uint64, index uint64) ([]*ethpb.Attestation, error) {
+	s.attestationPoolLock.RLock()
```

Comment:
Since this is locking, can you add a span before acquiring the lock? It would help highlight lock blocking in jaeger